Python/DocumentationUtilsPackage/DocumentationUtils/__init__.py

```diff
@@ -0,0 +1 @@
+
from DocumentationUtils import CommandsUtils
from mdutils.mdutils import MdUtils


#%%

def generateShapeWorksCommandDocumentation(mdFilename = '../../Documentation/ShapeWorksCommands/ShapeWorksCommands.md'):
    # settings from Executable.cpp
    opt_width  = 32
    indent     = 2
    spacedelim = ''.ljust(indent)
    
    mdFile     = MdUtils(file_name = mdFilename, title = 'ShapeWorks Commands')
    cmd        =  "shapeworks"
    CommandsUtils.addCommand(mdFile, cmd, level = 1, spacedelim = spacedelim, verbose = True)
    mdFile.new_table_of_contents(table_title='Contents', depth=2)
    mdFile.create_md_file()
    
#%%


```

Comment:
Possible issue this file is filled with ^M instead of proper newlines. On GitHub just looks like one big line, mostly a comment.

Reply:
Maybe this is a Spyder thing. Not sure if this could be resolved automatically. I will open py files on GitHub as @cchriste  did.

Reply:
I just committed an updated init file but no ^M were written. I wonder why it was there in the first place :(